content/courses/advanced-ada/chapters/numerics.rst

```diff
@@ -1640,10 +1640,15 @@ Big Numbers
 -----------
 
 As we've seen before, we can define numeric types in Ada with a high degree of
-precision. However, in certain applications, even that precision isn't enough,
-so we have to rely on
+precision. However, these normal numeric types in Ada are limited to what
+the underlying hardware actually supports. For example, any signed integer
+type |mdash| whether defined by the language or the user |mdash| cannot have a
+range greater than that of :ada:`System.Min_Int .. System.Max_Int` because
+those constants reflect the actual hardware's signed integer types. In certain
+applications, that precision might not be enough, so we have to rely on
 :wikipedia:`arbitrary-precision_arithmetic <arbitrary-precision_arithmetic>`.
-Ada's big numbers allow us to operate with this kind of arithmetic.
+Ada's big numbers, which are not constrained by the underlying hardware, allow
+us to operate with this kind of arithmetic.
```

Comment:
Probably ", allow us to operate" should be ", allowing us to operate" or something like that.